doc/smocodesample/CoreApp/ModifySetting/project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable",
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "sharedmanagementobjects": "(140.0.0,150.0.0)"
```

Comment:
>140.0.0,150.0.0 [](start = 33, length = 15)

I believe there is a syntax that specifies any version of 140.0.0, but not 150.0.0.  I think that's the syntax we want to use when referencing the 140 SMO SDK.